spec/features/user_votes_on_reviews_spec.rb

```diff
@@ -3,30 +3,84 @@
 require 'rails_helper'
 
 feature 'User votes on reviews' do
-  let!(:user) { FactoryGirl.create(:user) }
-  let!(:recipe) { FactoryGirl.create(:recipe) }
   let!(:review) { FactoryGirl.create(:review) }
+  let!(:user) { review.user }
+  let!(:recipe) { review.recipe }
+  let!(:user2) { FactoryGirl.create(:user, first_name: "Jane") }
```

Comment:
Consider a more descriptive name than `user2`. 

Similarly, there are instructions in the `FactoryGirl` getting started guide for a quick handy shortcut so that you don't have to type out `FactoryGirl` every time!